src/components/Reports/Reports.js

```diff
@@ -17,17 +17,28 @@ import { addDays } from '../../utils';
 
 const periodSelectedDaysDefault = 7;
 
+const getDateNow = (intl) => {
+  return new Date(
+    intl.formatDate(Date.now(), {
+      year: 'numeric',
+      month: 'numeric',
+      day: 'numeric',
+    }),
+  );
+};
```

Comment:
I do not understand what is the meaning of this function?

Reply:
create the date only using year month and day with 0 minutes and 0 seconds

Reply:
There should be a better way without `intl` (because it has nothing to do with localization).

Also, this method could be along with `addDays`, maybe it could be called `beginingOfDay`. For example:

```js
var today = beginingOfDay(new Date());
var yesterday = addDays(today, -1);
```

Reply:
i changed